test/unit/tcache_max.c

```diff
@@ -1,4 +1,7 @@
 #include "test/jemalloc_test.h"
+#include "test/san.h"
+
+const char *malloc_conf = TEST_SAN_UAF_ALIGN_DISABLE;
```

Comment:
This is the only test not working with `san_uaf_align` -- it assumes and checks the exact caching behavior which is a bit hard to work around. Explicitly disables the uaf check option, as I envision it will become default on in the future (otherwise the test will be always skipped when the default changes).